test/cljs/rems/administration/test_create_catalogue_item.cljs

```diff
@@ -46,3 +46,82 @@
+                                :licensetype "text"
+                                :localizations {:en {:attachment-id nil
+                                                     :textcontent "Be fast."
+                                                     :title "Second License"}}}]
+                    :modifieruserid "owner"
+                    :organization {:organization/id "nbn"
+                                   :organization/name {:en "NBN"
+                                                       :fi "NBN"
+                                                       :sv "NBN"}
+                                   :organization/short-name
+                                   {:en "NBN en"
+                                    :fi "NBN fi"
+                                    :sv "NBN sv"}}
+                    :owneruserid "owner"}
+                   {:resid "y"
+                    :licenses [{:archived false
+                                :enabled true
+                                :id 8
+                                :licensetype "text"
+                                :localizations {:en {:attachment-id nil
+                                                     :textcontent "Be fast."
+                                                     :title "Performance License"}}}
+                               {:archived false
+                                :enabled true
+                                :id 8
+                                :licensetype "text"
+                                :localizations {:en {:attachment-id nil
+                                                     :textcontent "Be fast."
+                                                     :title "Second License"}}}]
+                    :organization {:organization/id "nbn"
+                                   :organization/name {:en "NBN"
+                                                       :fi "NBN"
+                                                       :sv "NBN"}
+                                   :organization/short-name {:en "NBN en"
+                                                             :fi "NBN fi"
+                                                             :sv "NBN sv"}}}
+                   {:resid "y"
+                    :licenses [{:archived false
+                                :enabled true
+                                :id 8
+                                :licensetype "text"
+                                :localizations {:en {:attachment-id nil
+                                                     :textcontent "Be fast."
+                                                     :title "Performance License"}}}]}
+                   {:resid "u"}
+                   {:resid "z"}]
+        counts (frequencies (map :resid resourses))]
+    (testing "sorting resources"
+      (is (not (empty? (map resourse-dropdown resourses))))
+      (is (= ["x (org: NBN en )"
+              "y (org: NBN en ) (licenses: Performance License, Second License )"
```

Comment:
Could you get rid of the extra space? I mean `(org: NBN en)` instead of `(org: NBN en )` and similarly for `(licenses: ...)`. Just something that bugs me.

Reply:
sure, it was bugging me too, but I wanted to very this with you